external/untwine/api/QgisUntwine_unix.cpp

```diff
@@ -15,6 +15,7 @@ bool QgisUntwine::start(Options& options)
 {
     int fd[2];
     int ret = ::pipe(fd);
+    (void)ret;
```

Comment:
can we backport this to https://github.com/hobu/untwine?

Reply:
https://github.com/hobu/untwine/pull/49